backend/pong/constants.py

```diff
@@ -0,0 +1,13 @@
+# constants.py
+PLAYER_MOVEMENT_UNIT = 1
+PLAYGROUND_WIDTH = 1000
+PLAYGROUND_HEIGHT = 400
+MESSAGE_INTERVAL_SECONDS = 0.05  # 50 milliseconds
+PLAYER_WIDTH = 1
+PLAYER_HEIGHT = 40
+BALL_WIDTH = 1
+BALL_HEIGHT = 1
+WALL_MARGIN = 2
+BALL_SPEED = 5 
+MAX_BOUNCE_ANGLE = 90  # Bounce angle in degrees
+MIN_BOUNCE_ANGLE = -90
```

Comment:
Is this a django thing, isnt it more favorable to use radians instead of degrees for angles as radians is more accurate?